src/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { startLogin } from 'store/auth/actions';
+import { Button } from 'styles/elements/Button';
+
+const LoginWrapper = styled.div``;
+
+const LoginPage = ({ onStartLogin }) => (
+  <LoginWrapper>
+    <Button onClick={onStartLogin}>Login</Button>
+  </LoginWrapper>
+);
+
+LoginPage.propTypes = {
+  onStartLogin: PropTypes.func.isRequired,
+};
+
+const mapDispatchtoProps = dispatch => ({
+  onStartLogin: () => dispatch(startLogin()),
```

Comment:
the same as before. By the way... you can use `useDispatch` ;) espacially in such a small function. 
in `mponents/Header/Header.js` as well